apps/flame-defi/app/components/navbar/navbar.tsx

```diff
@@ -65,14 +65,14 @@ const Navbar = () => {
           >
             SWAP
           </Link>
-          {featureFlags.poolEnabled && (
-            <Link
-              href="/pool"
-              className={navLinkClasses(pathname.startsWith("/pool"))}
-            >
-              POOL
-            </Link>
-          )}
+          {/* {featureFlags.poolEnabled && ( */}
```

Comment:
If helpful, you can set `NEXT_PUBLIC_FEATURE_POOL_ENABLED=true` in your `.env.local`

Reply:
I'll do that! Thanks